packages/webapp/src/pages/Home/HomePage.tsx

```diff
@@ -0,0 +1,11 @@
+/*!
```

Comment:
Is this component being used? If not, maybe we could remove it.

Reply:
Removed.